src/newViews/Tagging/Dashboard/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import Navbar from '../Navbar';
+
+function Dashboard() {
+    return (
+        <div>
+            <Navbar />
```

Comment:
Instead of keeping the Navbar in individual components, can we not put it in the main Tagging/index.tsx page?

Reply:
I believe this is the only we can put sub-navigation right now. @tnagorra any thoughts?